### DISCUSSION TITLE
How can I configure DRX parameters in srsRAN Project?

### DISCUSSION THREAD

**Discussion by goodsollee** [srsran/srsRAN_Project#847] [General Help] [answered]
Hi, I have several questions on configuring DRX in srsRAN Project.

1) Is DRX implemented in srsRAN Project?

2) If it is, how can I configure its parameters (e.g., drx inactivity timer) ?

Thanks in advance!

Best,
Goodsol.

**Comment by herlesupreeth** [comment, marked as answer]
> Is DRX implemented in srsRAN Project?

DRX in case of Idle is supported (not the connected mode DRX)

> If it is, how can I configure its parameters (e.g., drx inactivity timer) ?

```
cu_cp:
  inactivity_timer: 10
```

**Reply by goodsollee** [reply]
Thank you! I checked it.

It would be better if there is C-DRX, but idle mode DRX is also cool.
